src/SonarScanner.MSBuild.Shim/PropertiesFileGenerator.cs

```diff
@@ -480,5 +485,16 @@ private void FixEncoding(ProjectInfo projectInfo, string globalSourceEncoding)
                 }
             }
         }
+
+        private static bool IsInNuGetCache(string filePath)
```

Comment:
if it's only used above, make it a static local function?

it'll be easier to see that it's only used when file is outside of base dir

Reply:
Based on t[his](https://github.com/SonarSource/sonar-scanner-msbuild/pull/1256#discussion_r897878105) comment, now it's just one line check. There's no need for a separate method anymore.